src/background/Ground.hpp

```diff
@@ -14,7 +14,7 @@ struct GroundType {
 class Ground {
  public:
   Ground(sf::Texture& texture);
-  void draw(sf::RenderWindow& window, const int x, const int y);
+  void draw(sf::RenderWindow& window, int x, int y);
```

Comment:
Does clang-tidy complains about consts? :| I though it's considered to be a good practice to add consts in arguments

Reply:
It complains only about consts in declarations, because they do not matter. I left all the consts in definitions as they were before